### ISSUE TITLE
Can not login with Xshell, but putty can login well

### ISSUE THREAD

**Issue by tsjsdbd** [cloudfoundry-incubator/diego-ssh#6]
when use Xshell4, it always says: SSH server refused.

Baidu says the sshd config `/etc/ssh/sshd_conf` should configured like:
`PasswordAuthentication yes`

so which part of code we can check? proxy or daemon (will run in container)

thanks @shashidharatd

**Comment by ematpl** [comment]
Hi, @tsjsdbd,

Your SSH client will be communicating with the Diego SSH proxy, so I would look in its logs about why the connection is refused. Verbose logs from your client may also help identify the issue.

Also, in the time since you reported this issue, we've changed the password used to authenticate to CF app instances from the arbitrarily long UAA-issued access token to a much shorter UAA-issued one-time authorization code. We had observed that some SSH clients truncated the long access token at about 1024 bytes, which caused some users to be unable to authenticate. Are you still seeing this issue with the Xshell4 client when using this new authentication method?

Thanks,
Eric, CF Runtime Diego PM
